src/components/ContactForm/ContactForm.js

```diff
@@ -0,0 +1,105 @@
+import { useState } from 'react';
+import Button from '../Button/Button';
+import FormInput from '../FormInput/FormInput';
+import styled from 'styled-components';
+import emailjs from 'emailjs-com';
+
+export default function ContactForm2() {
+  const [values, setValues] = useState({
+    restaurantname: '',
+    address: '',
+    email: '',
+    telephone: '',
+  });
+
+  const inputs = [
+    {
+      id: 1,
+      name: 'restaurantname',
+      type: 'text',
+      errorMessage: 'Should not include any special characters',
+      label: 'Restaurantname',
+      required: true,
+      pattern: /[^A-Za-z0-9\- .]/g,
+    },
+    {
+      id: 2,
+      name: 'address',
+      type: 'text',
+      errorMessage: 'Address should include Street and Number',
+      label: 'Address',
+      required: true,
+      //  pattern: /[^A-Za-z\- . [0-9]$]/g,
+    },
+    {
+      id: 3,
+      name: 'email',
+      type: 'email',
+      errorMessage: 'It should be a valid email address',
+      label: 'E-Mail',
+      required: true,
+      // pattern:
+      //   /^[a-zA-Z0-9][-.+!#$%&'*/=?^`{|]{0,1}([a-zA-Z0-9][-.+!#$%&'*/=?^`{|]{0,1})[a-zA-Z0-9]@[a-zA-Z0-9][-.]{0,1}([a-zA-Z][-.]{0,1})[a-zA-Z0-9].[a-zA-Z0-9]{1,}([.-]{0,1}[a-zA-Z]){0,}[a-zA-Z0-9]{0,}$/i,
```

Comment:
If you don't use something, please delete it and delete comments. We always try to have the `main` branch as clean as possible.